driver/http/BUILD

```diff
@@ -0,0 +1,24 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package(default_visibility = ["//visibility:public"])
+
+load("@typedb_dependencies//tool/checkstyle:rules.bzl", "checkstyle_test")
+
+files = [
+    "http.feature",
```

Comment:
Good idea, we can definitely also test http in different languages and from the server too!